python/lsst/ts/athexapod/controller.py

```diff
@@ -23,6 +23,9 @@
 
 import asyncio
 import logging
+from typing import Optional
```

Comment:
Please do not use `Optional` anymore. Also do not use `salobj.State` but import `sal_enums` from ts_xml and use `sal_enums.State` instead. So this would simply become `sal_enums.State | int`. See https://docs.python.org/3.12/library/typing.html#typing.Optional Please do this everywhere else as well.